astropy/cosmology/core.py

```diff
@@ -777,6 +791,46 @@ def nu_relative_density(self, z):
 
         return prefac * self._neff_per_nu * rel_mass
 
+    def _get_nufunc_scalar(self):
+        """ Return the version of the nu relative density function
+        to use in integrals."""
+
+        if self._massivenu:
+            return self._massivenu_relative_density_scalar
+        else:
+            return lambda x : 0.22710731766 * self._Neff
```

Comment:
If I had to guess it's just nice to have the implementation right here, rather than off in another file (unless the entire class were written in Cython which would also be pretty doable in this case...)